File: LENTABP10/src/main/java/com/lenta/lentabp10/models/task/TaskType.java
package com.lenta.lentabp10.models.task;

import lombok.Getter;

public class TaskType {

    @Getter private String code;
    @Getter private String name;

    public TaskType(String code, String name) {
        this.code = code;
        this.name = name;
    }
}

File: writeoff_breakage/src/main/java/com/lenta/bp10/models/task/TaskType.kt
package com.lenta.bp10.models.task

class TaskType(val code: String,val name: String){

    override fun equals(other: Any?): Boolean {
        if (this === other) return true
        if (other?.javaClass != javaClass) return false
        other as TaskType

        return code == other?.code
    }
}
